server/src/mindustry/server/ServerControl.java

```diff
@@ -172,6 +172,18 @@ public ServerControl(String[] args){
             }
         });
 
+        Events.on(WaveEvent.class, e -> {
+            if(state.is(State.playing) && Config.valueOf("enableAutosave").bool() ){
+                Fi file = saveDirectory.child("autosave-" + Strings.capitalize(world.getMap().name()) + "-" + state.wave + "." + saveExtension);
+
+                Core.app.post(() -> {
+                    SaveIO.save(file);
+                    Call.sendMessage("[scarlet][Server]:[] - Autosave Done");
```

Comment:
I don't think players should know or care about the autosave. Don't send a message about it.